<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BirthdayAstroWidget</class>
 <widget class="QWidget" name="BirthdayAstroWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>星座查询器</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_year">
       <property name="text">
        <string>年份：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="yearEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_month">
       <property name="text">
        <string>月份：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="monthEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_day">
       <property name="text">
        <string>日期：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="dayEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="queryButton">
     <property name="text">
      <string>查询星座和年龄</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="resultLabel">
     <property name="text">
      <string>结果将在此显示</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>